ui/src/features/stage/tabs/freight-history/freight-history.tsx

```diff
@@ -92,78 +92,76 @@ export const FreightHistory = ({
-                  setSelectedRequestedFreight(newRequestedFreight);
-                }
-              }}
-              options={requestedFreights?.map((i) => ({
-                label: i.origin?.name,
-                value: `${i?.origin?.kind}/${i?.origin?.name}`
-              }))}
-            />
-          </Descriptions.Item>
-        </Descriptions>
-
-        <Table
-          dataSource={freightReferences}
-          size='small'
-          pagination={{ hideOnSinglePage: true }}
-          rowKey={(record, index) => `${record.name}${index}`}
-        >
-          <Table.Column<PlainMessage<FreightReference>>
-            title='ID'
-            width={100}
-            render={(value, record) =>
-              freightMap[record?.name || '']?.metadata?.name?.substring(0, 7)
-            }
-          />
-          <Table.Column<PlainMessage<FreightReference>>
-            title='Alias'
-            width={240}
-            render={(value, record, index) => (
-              <Space>
-                <Link
-                  to={generatePath(paths.freight, {
-                    name: projectName,
-                    freightName: record.name
-                  })}
-                >
-                  {freightMap[record?.name || '']?.alias || record.name}
-                </Link>
-                {index === 0 && <Tag color='success'>Active</Tag>}
-              </Space>
-            )}
+      <Descriptions bordered className='max-w-md' size='small'>
+        <Descriptions.Item label={selectedRequestedFreight?.origin?.kind}>
+          <Select
+            value={freightUniqueIdentifier}
+            className='w-full'
+            onChange={(value) => {
+              const [kind, name] = value.split('/');
+              const newRequestedFreight = requestedFreights.find(
+                (i) => i.origin?.kind === kind && i.origin?.name === name
+              );
+
+              if (newRequestedFreight) {
+                setSelectedRequestedFreight(newRequestedFreight);
+              }
+            }}
+            options={requestedFreights?.map((i) => ({
+              label: i.origin?.name,
+              value: `${i?.origin?.kind}/${i?.origin?.name}`
+            }))}
           />
-          <Table.Column<PlainMessage<FreightReference>>
-            title='Contents'
-            render={(value, record) => (
-              <FreightContents
-                horizontal
-                fullContentVisibility
-                highlighted={false}
-                freight={create(FreightSchema, {
-                  metadata: {
-                    name: record.name
-                  },
-                  ...record
+        </Descriptions.Item>
+      </Descriptions>
+
+      <Table
+        dataSource={freightReferences}
+        size='small'
+        pagination={{ hideOnSinglePage: true }}
+        rowKey={(record, index) => `${record.name}${index}`}
+      >
+        <Table.Column<PlainMessage<FreightReference>>
```

Comment:
I think thats deprecated, could we just move this to `items` in table component as part of this PR?

Reply:
Why do you think it is deprecated?